src/content/docs/linter/index.mdx

```diff
@@ -227,35 +261,55 @@ You can set them by shaping the value of the rule differently.
 }
 ```
 
+### Domains
 
-## Linter Groups
+Domains are a Biome feature that allows to have more control over the lint rules. Domains could be seen as "concepts", or another form of category, however they offer different features compared to groups.
 
-The linter divides rules under _groups_. Groups are meant to offer some sort of category which rules falls under. This information becomes useful, for users, when choosing a rule to enable/disable, or for developers when creating new lint rules.
+A domain:
+- Can enable particular rules when Biome detects certain dependencies in the `package.json` of a project.
+- Can add particular global variables.
+- Can contains a small set of recommended rules.
 
-### Accessibility
+Biome linter will enable the rules that belong to a domain when it detects certain dependencies in the nearest `package.json`. For example, if the `mocha` dependency is detected, Biome will enable the **recommended rules** of the [`test`](/linter/domains#test) domain.
 
-Rules focused on preventing accessibility problems.
+However, if there's no `package.json` or the default configuration doesn't apply, you can enable the domain via configuration:
 
-### Complexity
-Rules that focus on inspecting complex code that could be simplified.
+```json title="biome.json" ins={4}
+{
+  "linter": {
+    "domains": {
+      "test": "on"
```

Comment:
I'm getting this feeling it might be nicer if `"test"` were called `"testing"` instead :thinking:

Reply:
I don't have strong feelings, we're still in time :)